app/services/shopify_import/importers/base_importer.rb

```diff
@@ -0,0 +1,31 @@
+module ShopifyImport
+  module Importers
+    class BaseImporter
+      def self.import!(**params)
```

Comment:
Is there any particular reason to have this as a "bang" method? Why is it called `import!` instead of `import`? The use of bang shouldn't be to symbolize that this method does something which is "important". It's usually used to symbolize that this method either modifies the original object (which is not possible, since it's a class method in this case) or that it will raise an error if the "happy path" is not possible for this method.

Please rename.